### PR TITLE
NEST-536: Update OC version

### PR DESCRIPTION
**Pull request by sarahelsaig**
[NEST-536](https://lombiq.atlassian.net/browse/NEST-536)

### CODE REVIEW

File: src/Modules/OrchardCore.Commerce/Startup.cs [hunk 1]
```diff
@@ -256,18 +257,15 @@ public class DeploymentStartup : StartupBase
...
-            });
-        services.AddSingleton<IDeploymentStepFactory>(new SiteSettingsPropertyDeploymentStepFactory<RegionSettings>());
+        services.AddDeployment<SiteSettingsPropertyDeploymentSource<RegionSettings>, SiteSettingsPropertyDeploymentStep<RegionSettings>>();
+        services.AddScoped(ImplementationFactory);
+    }
+
+    [SuppressMessage("Naming Rules", "SA1312:Variable names should begin with lower-case letter", Justification = "It's the IStringLocalizer.")]
+    private IDisplayDriver<DeploymentStep> ImplementationFactory(IServiceProvider serviceProvider)
+    {
+        var T = serviceProvider.GetService<IStringLocalizer<DeploymentStartup>>();
```
Comment (Piedone) [inline src/Modules/OrchardCore.Commerce/Startup.cs:267]: Why not just name it `localizer`?

**Review by Piedone — CHANGES_REQUESTED** [review]

File: src/Modules/OrchardCore.Commerce/Startup.cs (hunk 1 above)
Comment (sarahelsaig) [reply src/Modules/OrchardCore.Commerce/Startup.cs:267]: It was already like that above, I just moved the code around. Also I think `localizer` is not picked up by the PO Extractor. (although it really should, let me check if maybe there is already an issue for that).

**Review by Piedone — APPROVED** [review]

**Review by Piedone — APPROVED** [review]

**Review by Piedone — APPROVED** [review]

File: src/Modules/OrchardCore.Commerce/Startup.cs [hunk 2]
```diff
@@ -258,18 +258,16 @@ public class DeploymentStartup : StartupBase
...
-            });
-        services.AddSingleton<IDeploymentStepFactory>(new SiteSettingsPropertyDeploymentStepFactory<RegionSettings>());
+        services.AddDeployment<SiteSettingsPropertyDeploymentSource<RegionSettings>, SiteSettingsPropertyDeploymentStep<RegionSettings>>();
+        services.AddScoped(ImplementationFactory);
+    }
+
+    private IDisplayDriver<DeploymentStep> ImplementationFactory(IServiceProvider serviceProvider)
+    {
+        var localizer = serviceProvider.GetService<IStringLocalizer<DeploymentStartup>>();
+        return new SiteSettingsPropertyDeploymentStepDriver<RegionSettings>(
```
Comment (hishamco) [inline src/Modules/OrchardCore.Commerce/Startup.cs:268]:
```suggestion

        return new SiteSettingsPropertyDeploymentStepDriver<RegionSettings>(
```

**Review by hishamco — CHANGES_REQUESTED** [review]

**Review by hishamco — APPROVED** [review]

**Comment by hishamco** [comment]
@sarahelsaig could you please enable auto deleting for the branches after merge

**Pull request merged**